### PR TITLE
trying to fix import at docstring

### PR DESCRIPTION
**Pull request by arnaldorusso**
Maybe it fixes #26 Travis failure
At least, here locally it passes!

### CODE REVIEW

File: oceans/datasets/datasets.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ def woa_subset(llcrnrlon=2.5, urcrnrlon=357.5, llcrnrlat=-87.5, urcrnrlat=87.5,
     >>> import numpy.ma as ma
     >>> import matplotlib.pyplot as plt
     >>> from oceans.colormaps import cm, get_color
+    >>> from oceans.datasets import map_limits, woa_subset
     >>> from mpl_toolkits.basemap import Basemap
```
Comment (ocefpaf) [inline oceans/datasets/datasets.py]: The failure is unrelated to this.  Like I suspected, it was a server issue that seems to be fixed now.  I can also run it locally without the changes and soon we'll know if travis-ci can run it too.

However, for completeness of the example, the changes are welcome.  Thanks.

**Comment by ocefpaf** [comment]
And now that travis is done we can see that the issue persists :disappointed: 

https://travis-ci.org/pyoceans/python-oceans/jobs/54083726#L553

I suspect some connectivity limitation, travis might be trying to save bandwidth and netCDF cannot download the variables.  I will try to make a slimmer example/doctest later.

**Comment by arnaldorusso** [comment]
maybe including such .nc file inside a `data` folder, until you include the migration to iris dataset.

btw, where I can find the install instructions to travis? `.travis.yml` I got the list of packages to be installed inside setup.py [only pytest and pytest-cov]
It's a good idea to include the file `requirements.txt` and `dev-requirements.txt`? or some instructions for deving proccess inside any document?

Cheers.

**Comment by ocefpaf** [comment]
> maybe including such .nc file inside a data folder

I am reluctant to bloat the repo with binaries. Anyways, oceans is such a small project that we can just live with that :grimacing: 

> where I can find the install instructions to travis?

[`.travis.yml`](https://github.com/pyoceans/python-oceans/blob/master/.travis.yml#L16) this line install everything.

> It's a good idea to include the file requirements.txt and dev-requirements.txt?

Yes.

>  or some instructions for deving proccess inside any document?

Also, yes :grinning:

**Pull request merged**